contracts/Comptroller/ComptrollerInterface.sol

```diff
@@ -1,10 +1,6 @@
+// SPDX-License-Identifier: BSD-3-Clause
 pragma solidity 0.8.25;
 
-import { ResilientOracleInterface } from "@venusprotocol/oracle/contracts/interfaces/OracleInterface.sol";
-
-import { VToken } from "../Tokens/VTokens/VToken.sol";
-import { VAIControllerInterface } from "../Tokens/VAI/VAIControllerInterface.sol";
-
 enum Action {
```

Comment:
This enum can be moved to the `IFacetBase` since `actionPaused(address market, Action action)` was moved there. 

Then this file can be removed.